tests/snmp/test_snmp_loopback.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import time
+import logging
+import ipaddress
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+@pytest.mark.bsl
+def test_snmp_loopback(duthosts, rand_one_dut_hostname, nbrhosts, tbinfo, localhost, creds):
+    """
+    Test SNMP query to DUT over loopback IP 
+      - Send SNMP query over loopback IP from one of the BGP Neighbors
+      - Get SysDescr from snmpfacts 
+      - compare result from snmp query over loopback IP and snmpfacts
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    hostip = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars['ansible_host']
+    snmp_facts = localhost.snmp_facts(host=hostip, version="v2c", community=creds["snmp_rocommunity"])['ansible_facts']
+    config_facts = duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
+    # Get first neighbor VM information
+    nbr = nbrhosts[list(nbrhosts.keys())[0]]
+
+
+    # Copy config_service_acls.sh to the DuT (this also implicitly verifies we can successfully SSH to the DuT)
```

Comment:
Again, referring to the other shell script in the comment.

Reply:
Removed usage of script.